src/Microsoft.VisualStudio.AppDesigner/ApplicationDesigner/DesignerTabButton.vb

```diff
@@ -141,6 +141,37 @@ Namespace Microsoft.VisualStudio.Editors.ApplicationDesigner
             End If
         End Sub
 
+        Protected Overrides Sub OnKeyDown(kevent As KeyEventArgs)
+            MyBase.OnKeyDown(kevent)
```

Comment:
I'm surprised we're handling OnKeyDown instead of the ProcessDialogKey, why is that?